### ISSUE TITLE
Tests aren't visible in test explorer in visual studio code

### ISSUE THREAD

**Issue by MPSwift** [ttutisani/Xunit.Gherkin.Quick#153]
Hi,
I'm using Visual Studio Code, .NET v8 and the latest version of this framework (v.4.5) but I can't make it to show the tests in the test explorer. 
I can see all the other xUnit test cases (see below structur "MyOtherUnitTestCases") but not the cucumber tests ("MyApiTestCases").

I write the cs file with the step definition where given, when, then contains simply a log "not yet implemented" and I have the feature file. 
My expectation is to see the tests in the test explorere and if I run them to get the log prints "not yet implemented".

The structur is like:
MyProject
--MyApiTestCases
----MyTest.feature
----MySteps.cs
--MyOtherUnitTestCases
----MyUnitTestA
----MyUnitTestB

What am I missing? Should feature file and cs file have the same name?

**Comment by ttutisani** [comment]
Yes, the feature file and cs file are linked using the [FeatureFile] attribute.

If this does not resolve the issue, since there are multiple steps to properly configuring the tests, I suggest that you follow the [Getting Started](https://github.com/ttutisani/Xunit.Gherkin.Quick) tutorial (on the home page). Once that works, modify to tailor to your needs.

**Comment by MPSwift** [comment]
Hm, the [FeatureFile] attribute exist and I followed the getting started but the tests are still not visible in the test explorer in VS Code. 
I tried with coping the Calculator.cs, the AddTwoNumbers.cs and AddTwoNumbers.feature but they are also not visible in the test explorer.

**Comment by ttutisani** [comment]
So the ProjectConsumer tests are not showing out of the box? Okay, that must be something else, then.

**Comment by tobiasjstafford** [comment]
This is probably way too late to be of use to you @MPSwift, but I’ve banged my head on the wall with this issue a few times myself. 

I’ve found that small errors in your spec file can actually cause this symptom. For instance, one time I was using Scenario Outlines, but forgot the Outline keyword. No errors visible in the editor, but the test failed to show up. 

Additionally, I’ve noticed that I need to rebuild the test project to get it to detect changes to the spec file (in case you're changing it and still not seeing it show up.)
